config/dev.exs

```diff
@@ -2,4 +2,13 @@ use Mix.Config
 
 config :ecto, Ecto.Ldap.TestRepo,
        adapter: Ecto.Ldap.Adapter,
-       url: "ldap+ssl://ldap.puppetlabs.com/dc=com,dc=puppetlabs,ou=users"
+       url: "ldap+ssl://ldap.puppetlabs.com/dc=com,dc=puppetlabs,ou=users",
+       pool_size: 1
+
+config :exldap, :settings,
+       server: "LDAP Server goes here",
+       base: "dc=puppetlabs,dc=com",
+       port: 636,
+       ssl: true,
+       user_dn: "user_goes_here",
```

Comment:
I was super paranoid about anything internal following along to a public repo, so I opted to remove all traces to be safe.